sources/mapzenborders.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "Mapzen Borders",
 	"name": "mapzenborders",
 	"prefix": "mzb",
-    	"key": "id",
+	"key": "id",
 	"url": "https://mapzen.com/data/borders/",
-	"license": "CC0",
-	"description": ""
-}
+	"license": "https://mapzen.com/terms/",
+	"license_type": "CC0",
+	"license_text": "The person who associated a work with this deed has dedicated the work to the public domain by waiving all of his or her rights to the work worldwide under copyright law, including all related and neighboring rights, to the extent allowed by law. You can copy, modify, distribute and perform the work, even for commercial purposes, all without asking permission.",
+	"description": "Country, region and city boundary data from OpenStreetMap, served monthly."
```

Comment:
The last run of this was Oct. 2016. So instead of saying `served monthly` we could say `served monthly until October 2016.`